flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBOperationUtils.java

```diff
@@ -123,16 +125,36 @@ public static RocksDBKeyedStateBackend.RocksDbKvStateInfo createStateInfo(
             RocksDB db,
             Function<String, ColumnFamilyOptions> columnFamilyOptionsFactory,
             @Nullable RocksDbTtlCompactFiltersManager ttlCompactFiltersManager,
-            @Nullable Long writeBufferManagerCapacity) {
+            @Nullable Long writeBufferManagerCapacity,
+            List<ExportImportFilesMetaData> cfMetaDataList) {
```

Comment:
Can't we use an empty list for `cfMetaDataList` instead of `null`?
If not, mark `@Nullable`?